src/csharp/BUILD-INTEGRATION.md

```diff
@@ -91,6 +91,14 @@ button](https://stackoverflow.com/a/9770061). Click on it, and choose "Add as
 link". If you do not select this option, Visual Studio will copy files to the
 project directory instead.
 
+Alternatively, you can also specify `AdditionalImportDirs` to give the location of proto files outside
```

Comment:
rephrase "to give the location of proto files outside
the project" as e.g. "to provide the list of directories in which the protocol buffer compiler will search for imported proto files".
(your original phrasing seems a bit vague).

Reply:
updated